React/Base/RCTDevMenu.h

```diff
@@ -47,6 +47,24 @@
 
 @end
 
+@interface RCTDevMenuConfiguration : NSObject {
+  NSTimeInterval liveReloadPeriod;
+  BOOL liveReloadEnabled;
+}
+
+@property (nonatomic, assign) NSTimeInterval liveReloadPeriod;
+@property (nonatomic, assign) BOOL liveReloadEnabled;
+
++ (RCTDevMenuConfiguration *)sharedInstance;
+
++ (NSTimeInterval)getLiveReloadPeriod;
++ (BOOL)getLiveReloadEnabled;
+
++ (void)setLiveReloadPeriod:(NSTimeInterval)liveReloadPeriod;
++ (void)setLiveReloadEnabled:(BOOL)liveReloadEnabled;
```

Comment:
Yeah I'm not sure how great of an idea that is but it seemed like the simplest way to store config independent of the `RCTDevMenu` instance. Diff above looks out of date still, I removed the redundant property declarations.